modules/network/src/integrationTest/java/org/apache/ignite/utils/ClusterServiceTestUtils.java

```diff
@@ -44,30 +45,32 @@
      *
      * @param nodeName Local name.
      * @param port Local port.
-     * @param nodeFinder Node finder for discovering the initial cluster members.
+     * @param nodeFinderConfigurer Node finder configurer.
      * @param msgSerializationRegistry Message serialization registry.
      * @param clusterSvcFactory Cluster service factory.
      */
     public static ClusterService clusterService(
         String nodeName,
         int port,
-        NodeFinder nodeFinder,
+        Consumer<NodeFinderChange> nodeFinderConfigurer,
```

Comment:
As I said above, I don't like this approach. I would prefer using a `NodeFinder` instance as before.